ibft/storage/store.go

```diff
@@ -138,12 +137,12 @@ func (i *ibftStorage) getParticipants(txn basedb.ReadWriter, identifier convert.
 	return operators, nil
 }
 
-func (i *ibftStorage) saveParticipants(txn basedb.ReadWriter, identifier convert.MessageID, slot phase0.Slot, operators []spectypes.OperatorID) error {
+func (i *participantStorage) saveParticipants(txn basedb.ReadWriter, role spectypes.BeaconRole, pk spectypes.ValidatorPK, slot phase0.Slot, operators []spectypes.OperatorID) error {
 	bytes, err := encodeOperators(operators)
 	if err != nil {
 		return fmt.Errorf("encode operators: %w", err)
 	}
-	if err := i.save(txn, bytes, participantsKey, identifier[:], uInt64ToByteSlice(uint64(slot))); err != nil {
+	if err := i.save(txn, bytes, participantsKey, pk[:], uInt64ToByteSlice(uint64(role)), uInt64ToByteSlice(uint64(slot))); err != nil {
```

Comment:
@moshe-blox we use `pk[:]` in combination with `role` as key here, instead of the previous `MessageID` based identifier, wdyt?

Reply:
@anatolie-ssv this looks good, as long as we don't care to preserve the same keys as before (which is likely the case now)